flink-read-write-redis/main.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import numpy as np
+import pandas as pd
+from feathub.common import types
+from feathub.feathub_client import FeathubClient
+from feathub.feature_tables.sinks.redis_sink import RedisSink
+from feathub.feature_tables.sources.file_system_source import FileSystemSource
+from feathub.feature_tables.sources.redis_source import RedisSource
+from feathub.feature_views.on_demand_feature_view import OnDemandFeatureView
+from feathub.table.schema import Schema
+from pandas._testing import assert_frame_equal
```

Comment:
`pandas._testing` looks like a private package that is expected to be used only inside pandas.

Would it be simpler to just raise an exception if `expected_features.equals(online_features)` returns false?